corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -28,6 +29,23 @@ var eventize = function (that) {
     return that;
 };
 
+// This is copy of csrf ajax utilities from hqwebapp/js/ajax_csrf_setup.js
```

Comment:
Wait, I already did that. See [`ajax_csrf_setup.js`](https://github.com/dimagi/commcare-hq/pull/9227/files#diff-10bef7ead42aafa580f5d7973b364700R6) which is being reused in lot of places.

This situation is little more complicated in that, the `main.js` here is kind of an API for saveButton which is used in lot of places. The options I have here are either 1. load above reusable ajax_csrf_setup.js in all those places where `main.js` is loaded or 2. Copy paste ajax_csrf_setup to `main.js` so that that repetition is avoided.